first draft for hackathon website (#1202)
nf-core/nf-co.re: markdown/events/2022/hackathon-october-2022.md

> 

Author: your changes did not say that the registration is open, but that it will be open in early July. Want me to change that?

> In addition to the in-person event, we will endeavour to run a hybrid option online in [gather.town](https://gather.town/).

Reviewer: Sorry, I know you didn't touch this bit but as you're here anyway.. Maybe can move this into the first paragraph. eg. something like:

```
This hackathon will be held in advance of the _**Nextflow Summit 2022**_ in Barcelona, Spain 🇪🇸 and online in  [Gather.town](https://gather.town/). Find out more information about the Nextflow Summit, please visit <https://summit.nextflow.io>. 
```

> This hackathon is not meant to be a training event but aimed at people that already have a basic understanding of nextflow and/or nf-core. For anyone interested in learning nextflow, there are going to be workshops in the week before the hackathon. As soon as we have more information, we will let you know.

Reviewer 2: Are they happening? The summit is after the hackathon iirc

Author: > Are they happening?
That was what I understood from our discussion last Friday at least. Maybe we can wait until the next meeting to ask again?

> The summit is after the hackathon iirc
Hackathon first, then summit, right? That's what I say, I believe?

Reviewer 2: yes ok that is why I was wondering about the workshop before the hackathon. Or is that yet anothe event?

Author: no, it's the workshops we talked about and that are mentioned in the meeting notes.

Reviewer: Suggestion updated 👍🏻

Reviewer: x-ref: https://github.com/nf-core/nf-co.re/pull/1203

> <!--
> 
> # Registration
> 
> Registration will open soon. Please check back here and keep an eye out on twitter and Slack,
> but in the mean time - book the dates in your calendar!
> 
> -->

Reviewer: ```suggestion
# Registration

Registration is now open together with the Nextflow Summit: <https://summit.nextflow.io/register> 

You can register for either the hackathon or the summit, or both. You're welcome to attend in-person or online.
Attendance online is free, attending in person costs €49 (€59 incl. VAT).

Registration will close September 9th, 2022.

<a class="btn btn-success btn-lg" href="https://summit.nextflow.io/register/">Register now</a>
```

> # Topic: Free-hacking!
> 
> This time around we decided to not have an official topic. You are welcome to join the hackathon and work on your own pipelines or help improve existing ones. Of course you are also welcome to add to documentation!

Reviewer: Topics was a bit of an unusual thing, so I don't think that we really need to point out that we're not doing that again.

```suggestion
# Groups
```

> To keep things manageable, we will organise into groups.

Reviewer: ```suggestion
To keep work and discussion during the hackathon manageable, we will organise everyone into groups.
```

You are free to change groups as you wish during the hackathon.

> Since the nextflow summit is going to be held just after the hackathon, we decided to keep talks to a minimum. There will however be some workshops that include talks. We keep you posted on that!

Reviewer: ```suggestion
Since the Nextflow Summit is going to be held just after the hackathon, we decided to keep presentations to a minimum. There will however be some workshops. More details about those will be given closer to the event.
```

> We will coordinate our work and the issues we are working on using a single GitHub which we will post a link to here closer to the date. There will be a separate tab for each group where will have collated a list of relevant issues.

Reviewer 2: ```suggestion
We will coordinate our work and the issues we are working on using a single GitHub Project Board which we will post a link to here closer to the date. There will be a separate tab for each group where will have collated a list of relevant issues.
```

Reviewer: hah, I just did a suggested change to drop these caps 😆 Ah well, potato Potato...

> We will coordinate our work and the issues we are working on using a single GitHub Project Board which we will post a link to here closer to the date. There will be a separate tab for each group where will have collated a list of relevant issues.

Reviewer: ```suggestion
We will coordinate our work and the issues we are working on using a single GitHub _project board_ (we'll post a link to here closer to the time). There will be a separate tab for each group where will have collated a list of relevant issues.
```

## Group 1 - Documentation

> This group will focus on all kind of documentation, from pipelines themselves to modules and aid in getting started with nf-core and nextflow. We are also planning on making intro videos for each pipeline.

Reviewer: ```suggestion
This group will focus on all kind of documentation, from pipelines themselves to modules and general documentation to aid in getting started with nf-core and Nextflow. We are also planning on making introduction videos for each pipeline.
```

> ## Group 2 - Subworkflows
> 

Reviewer: ```suggestion
## Group 2 - Subworkflows

Why stop at sharing modules, we want to share subworkflows between pipelines too! Join this group to work on the cutting edge of nf-core development, fleshing out details about how this functionality will work and building proof of concept code to use as a model for the rest of nf-core. Recommended for relatively experienced Nextflow / nf-core contributors only.

```

## Group 3 - Pipelines

This group will focus on any pipeline related work, this could be:

- DSL2 conversion of a pipeline

> - new / improved features
> - new pipelines

Reviewer: ```suggestion
- New / improved features for a pipeline
- Entirely new pipelines
```

## Group 4 - Modules

> This group will welcome anyone who wants to work on new Modules or improve already existing ones.

Reviewer: ```suggestion
This group will welcome anyone who wants to work on new modules or improve existing ones.
```

## Group 5 - Central tools / Website

The nf-core website needs to be improved and maintained. If this sounds like a fun task, join group 5!

# Schedule

> TBD

Reviewer: ```suggestion
Coming soon..
```
